lib/realtime/vehicles.ex

```diff
@@ -58,7 +58,6 @@ defmodule Realtime.Vehicles do
     incoming_from_another_route = incoming_from_another_route(incoming_trips, vehicles_and_ghosts)
 
     vehicles_and_ghosts
-    |> Enum.filter(fn vehicle_or_ghost -> vehicle_or_ghost.route_id != nil end)
```

Comment:
This where logged out vehicles were originally being filtered out. Keeping them in results in a key `nil` being added to the result of `group_by_route_with_blocks`. Something about a nil key of a map is funky to me, but it seemed like the simplest way to keep these logged out vehicles for use elsewhere